src/appmetrics.cpp

```diff
@@ -285,7 +285,7 @@ static void* getMonitorApiFunction(std::string pluginPath, std::string functionN
     std::string libname = "hcapiplugin.dll";
 #elif defined(__MACH__) || defined(__APPLE__)
     std::string libname = "libhcapiplugin.dylib";
-#elif defined (__AIX__) || defined(AIX)
```

Comment:
@ab-m - what's the purpose of this change?

Reply:
@sjanuary `_AIX` is predefined on both AIX & IBM i PASE systems whereas `AIX` is not predefined on. In this instance both AIX & PASE would follow the same code path so `_AIX` was used.

Alternatively another `|| defined(__PASE__)` could have been appended to the line instead but the first approach using `_AIX` seemed cleaner.

Reply:
that's fine, I just wanted to understand the reason for it